fvm/errors/errors.go

```diff
@@ -114,6 +114,39 @@ func HandleRuntimeError(err error) error {
 		return NewUnknownFailure(err)
 	}
 
+	errs := []error{runErr}
```

Comment:
Maybe add a comment why this is needed, and link the isue for a cadence fix that would make this redundant if one exists.